tests/interactive/test_app.py

```diff
@@ -283,8 +289,14 @@ def callback(x: str):
         await pilot.pause()
         # assert after "Condense Button" is clicked that the state and condensed_pass list change accordingly
         assert app.condense_mode is True
+        rewrites = get_all_possible_rewrites(
+            expected_module,
+            individual_rewrite.REWRITE_BY_NAMES,
+        )
         assert app.available_pass_list == get_condensed_pass_list(
             expected_module, app.all_passes
+        ) + convert_indexed_individual_rewrites_to_available_pass(
+            rewrites, expected_module
```

Comment:
@superlopuh does this seem reasonable? I'm not sure I could think of a better way to do it.

Reply:
Yep I think so, this part of the test should verify that the signals get passed around successfully, so we just need to check the logic here.